new_account.php

```diff
@@ -23,27 +23,23 @@
 		<title>Graeme's Music</title>
 		
 		<link rel="stylesheet" href="css/style.css">
+		<link rel="icon" type="image/png" href="images/graeme_favicon_v2.png">
 	</head>
 	<body>
 		<!-- The header contains the logo, website name, and link to login or sign out -->
 		<div class="header">
 			<div class="logo">
-				<p>Logo placeholder</p>
+				<!-- Logo image that is also a link -->
+				<a href="index.php"><img src="images/graeme_music_logo_v2.png" alt="Goes to the main page"></a>
 			</div>
 			
 			<div class="header_text">
 				<h1>Graeme's Music</h1>
 			</div>
 			
 			<div class="user_info">
-				<p>User icon placeholder</p>
-				<?php
-				if(!isset($_SESSION['login_user'])) {
-					echo "<p><a href='login.php'>Login</a></p>";
-				} else {
-					echo "<p><a href='sign_out.php'>Sign out</a></p>";
-				}
-				?>
```

Comment:
This was a forgotten hold over from the contact page that didn't need to be here